Lzhl_tcp.cpp

```diff
@@ -54,7 +54,13 @@ SOCKET lzhl_socket( int af, int type, int protocol )
 
 SOCKET lzhl_accept( SOCKET s, struct sockaddr* addr, int* addrlen )
 {
+    // TheSuperHackers @feature vitimiti 13/03/2025 Added UNIX sockets support
+#if _WIN32
 	SOCKET sock = accept( s, addr, addrlen );
+#else
+	SOCKET sock = accept( s, addr, (socklen_t*)addrlen );
```

Comment:
On second look, is this safe? This is a `reinterpret_cast` and if `socklen_t` is larger than 4 bytes then this is an error.